spec/exchanges/novaexchange/integration/market_spec.rb

```diff
@@ -29,21 +29,4 @@
     it { expect(@ticker.timestamp).to_not be nil }
     it { expect(@ticker.payload).to_not be nil }
   end
-
```

Comment:
@dominicwong617 I did a quick check of the current response from Nova, indeed it has been removed, but they added another edge case: https://novaexchange.com/remote/v2/market/info/ESP2_DONATE_/. It seems like it's not an active market, should be safe to ignore.